libbeat/outputs/redis/docs/redis.asciidoc

```diff
@@ -23,7 +23,8 @@ output.redis:
 
 ==== Compatibility
 
-This output works with Redis 3.2.4.
+This output is expected to work with all Redis versions between 5.0.8 and 3.2.4. Other versions might work as well,
```

Comment:
I realize this is the backport PR which mirrors the PR to `master`, but any reason we listed the versions in this order? Ascending order seems more natural to me, but perhaps that's an English bias?

Reply:
I admit I missed it here, I will merge this PR and do another round of PRs to fix it.